patches/tbb.patch

```diff
@@ -0,0 +1,26 @@
+diff --git c/build/macos.inc i/build/macos.inc
+index dde93216..de912be8 100644
+--- c/build/macos.inc
++++ i/build/macos.inc
+@@ -38,6 +38,8 @@ ifndef arch
+  else
+    ifeq ($(shell /usr/sbin/sysctl -n hw.optional.x86_64 2>/dev/null),1)
+      export arch:=intel64
++   else ifeq ($(shell /usr/sbin/sysctl -n hw.optional.arm64 2>/dev/null),1)
++     export arch:=arm64
```

Comment:
aren't we explicitly defining the arch already though?

Reply:
Yep. I was trying to build this from an arm Mac with just make. This is not needed anymore since we're already specifying the arch when building.